packages/plugins/connection-manager/explorer/index.ts

```diff
@@ -245,12 +245,17 @@ export class ConnectionExplorer implements TreeDataProvider<SidebarTreeItem> {
   }
 
   public setActiveConnection(c: ConnectionInterface) {
-    Object.values(this.tree).forEach(item => {
```

Comment:
Ok, this is where I saw that was braking.

I should we move this code to a few lines below?

I didn't get the point of that.

Anyway I'm trying to reproduce to see the stack trace, but I don't remember exactly what I did that broke it :( lol

Reply:
Nvm. It was breaking because of the shadow vairable `item`. renaming fixes! Thanks @ariel-bentu!

I can see the performance enhancement in VSCode as well!